### PR TITLE
clang: fix license

### PR DESCRIPTION
**Pull request by 106062316**

### CODE REVIEW

**Comment by lazka** [comment]
@mati865

File: mingw-w64-clang/PKGBUILD [hunk 1]
```diff
@@ -36,23 +36,23 @@ pkgrel=3
 pkgdesc="C language family frontend for LLVM (mingw-w64)"
 arch=('any')
 url="https://llvm.org/"
```
Comment (lazka) [inline mingw-w64-clang/PKGBUILD:38]: bump pkgrel

**Comment by 106062316** [comment]
@lazka

**Comment by lazka** [comment]
If you change pkgver you also have to update all checksums (by calling `updpkgsums`)

**Comment by 106062316** [comment]
checksums updated

**Comment by 106062316** [comment]
@lazka is this good now?

**Pull request merged**